contracts/vaults/TreasuryVault/RibbonTreasuryVaultBare.sol

```diff
@@ -0,0 +1,1228 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.4;
+
+import {SafeMath} from "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {
+    SafeERC20
+} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {
+    ReentrancyGuardUpgradeable
+} from "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+import {
+    OwnableUpgradeable
+} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {
+    ERC20Upgradeable
+} from "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+
+import {Vault} from "../../libraries/Vault.sol";
+import {
+    VaultLifecycleTreasuryBare
+} from "../../libraries/VaultLifecycleTreasuryBare.sol";
```

Comment:
This file isn't uploaded, so we can either remove `RibbonTreasuryVaultBare` or add `VaultLifecycleTreasuryBare`. To prevent confusion for the future and better documentation maybe we can add `VaultLifecycleTreasuryBare` along with a comment in the contract `RibbonTreasuryVaultBare` on when it should be used and how it differs from the original.